src/test/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/ExtensibleChoiceParameterDefinitionJenkinsTest.java

```diff
@@ -627,14 +627,8 @@ public void testGetDefaultParameterValue() throws Exception
             job.getBuildersList().add(ceb);
             job.save();
             
-            try{
-                job.scheduleBuild2(job.getQuietPeriod()).get();
-                assertTrue("not reachable", false);
-            }
-            catch(IllegalArgumentException e)
-            {
-                assertTrue("non-editable, not in choice", true);
-            }
+            j.assertBuildStatusSuccess(job.scheduleBuild2(0));
+            assertEquals("non-editable, not in choice", "value4", ceb.getEnvVars().get("test"));

```

Comment:
Sorry, I mislead you. I meant expecting "value1" (the value at the top of the list).